charts/astronomer/values.yaml

```diff
@@ -158,6 +158,7 @@ prisma:
   #  requests:
   #   cpu: 100m
   #   memory: 128Mi
+  azureDb: false
```

Comment:
@ianstanton maybe we rename this `prisma.backendSecretName` to match Houston up above, and make a comment that if you are overriding the prisma secret, you have to use the right format / names for the keys.

At a minimum, this would just let us drop the `azure`-specific naming here.

Reply:
Pushed changes to reflect this